test/unit/spec/v1/Registration_spec.js

```diff
@@ -1037,4 +1037,126 @@ Expect.describe('Registration', function() {
         });
     });
   });
+  Expect.describe('sanitizes invalid request values', function() {
+    itp('null value', function() {
+      const preSubmitSpy = jasmine.createSpy('preSubmitSpy');
+      const setting = {
+        registration: {
+          preSubmit: function(postData, onSuccess, onFailure) {
+            preSubmitSpy(postData, onSuccess, onFailure);
+            onSuccess(postData);
+          },
+          postSubmit: jasmine.createSpy('postSubmitSpy'),
+        },
+      };
```

Comment:
Since this segment is reused a few times, is it possible for us to extrapolate it?

Reply:
Done